src/components/Publish/Pricing/Dynamic.tsx

```diff
@@ -78,6 +80,15 @@ export default function Dynamic({ content }: { content: any }): ReactElement {
           name="amountOcean"
           datatokenOptions={{ symbol: 'OCEAN', name: 'Ocean Token' }}
           weight={`${Number(weightOnOcean) * 10}%`}
+          min={
```

Comment:
what is the goal of this? Do we really need this logic here when we additionally validate with Yup?

Reply:
the purpose is to not let user entry a lower amount. The Yup validation would have shown an error but allow user to enter invalid ocean amount. 
Also if the ocean amount is invalid, the dataToken amount would become invalid too now that they are connected. So to prevent invalid dataToken amount too which will also need some messages I thought it would be better to limit the input to the min val.

Reply:
good thinking, but `min` and `max` only prevents going into those number ranges with the stepper arrows, not actual typing, they are there for HTML-only form validation which will never kick in with Formik form fields. 

When typing number outside of range, I get this:

<img width="890" alt="Screen Shot 2022-04-11 at 15 05 09" src="https://user-images.githubusercontent.com/90316/162756938-b5bda67a-3c76-4ac1-be32-a7effba77040.png">

Which to me looks sufficient and it's clear faulty values in other fields are based on this errored field, so that's already great with this PR!
 
So the only implication of using `min` here would be that stepper respects them, and the input `:valid/:invalid` pseudo classes might get set which could be used in CSS but we don't, as we use js validation and use those classes then. So having some logic in codebase, which duplicates a calculation we do somewhere else, and which in end we don't really use seems unnecessary. So would opt to remove this logic again

Reply:
came here to write that we need to move the Yup validation definition into the component to get access to dynamic values but I see you figured out a much better way already, great find with that custom validation method!